log/sequencer.go

```diff
@@ -24,6 +23,10 @@ type Sequencer struct {
 	keyManager crypto.KeyManager
 }
 
+// CurrentRootExpiredFunc examines a signed log root and decides if it has expired with respect
```

Comment:
I'd imagined a slightly different approach to this (not saying we should do that instead necessarily, but I thought I'd mention it); I was thinking that there would be Trillian STHs, which are produced with every single batch update to the tree (i.e. there's *always* a correct up-to-date signed root for the current state of the tree), and then there are application (i.e. CT) STHs, which are produced by an application "signer" which essentially just grabs the latest Trillian STH at whatever frequency is required, verifies it and then uses the tree hash/size/etc. to create a new application STH at the current time.